routes/cmip6_downscaled.py

```diff
@@ -0,0 +1,262 @@
+import asyncio
+import logging
+from flask import Blueprint, render_template, request
+
+# local imports
+from generate_urls import generate_wcs_query_url
+from generate_requests import generate_wcs_getcov_str
+from fetch_data import fetch_data, describe_via_wcps
+from validate_request import (
+    latlon_is_numeric_and_in_geodetic_range,
+    construct_latlon_bbox_from_coverage_bounds,
+    validate_latlon_in_bboxes,
+    project_latlon,
+    generate_time_index_from_coverage_metadata,
+)
+from postprocessing import postprocess, prune_nulls_with_max_intensity
+from csv_functions import create_csv
+
+from luts import cmip6_downscaled_options
+
+from . import routes
+
+logger = logging.getLogger(__name__)
+
+cmip6_api = Blueprint("cmip6_downscaled_api", __name__)
+
+all_possible_vars = list(cmip6_downscaled_options.keys())
+all_possible_models = set(
+    model for var in cmip6_downscaled_options.values() for model in var.keys()
+)
+all_possible_scenarios = set(
+    scenario
+    for var in cmip6_downscaled_options.values()
+    for model in var.values()
+    for scenario in model
+)
```

Comment:
This could get moved to `luts.py` if it appeals to your taste

Reply:
Fixed in fdfb744.